bundle/manifests/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -385,6 +385,63 @@ spec:
                         pattern: ^(Managed|Unmanaged|Force|Removed)$
                         type: string
                     type: object
+                  modelregistry:
+                    description: ModelRegistry component configuration.
+                    properties:
+                      devFlags:
+                        description: Add developer fields
+                        properties:
+                          manifests:
+                            description: List of custom manifests for the given component
+                            items:
+                              properties:
+                                contextDir:
+                                  default: ""
+                                  description: contextDir is the relative path to
+                                    the folder containing manifests in a repository
+                                  type: string
+                                sourcePath:
+                                  default: ""
+                                  description: 'sourcePath is the subpath within contextDir
+                                    where kustomize builds start. Examples include
+                                    any sub-folder or path: `base`, `overlays/dev`,
+                                    `default`, `odh` etc'
+                                  type: string
+                                uri:
+                                  default: ""
+                                  description: uri is the URI point to a git repo
+                                    with tag/branch. e.g  https://github.com/org/repo/tarball/<tag/branch>
+                                  type: string
+                              type: object
+                            type: array
+                        type: object
+                      managementState:
+                        description: "Set to one of the following values: \n - \"Managed\"
+                          : the operator is actively managing the component and trying
+                          to keep it active. It will only upgrade the component if
+                          it is safe to do so \n - \"Removed\" : the operator is actively
+                          managing the component and will not install it, or if it
+                          is installed, the operator will try to remove it"
+                        enum:
+                        - Managed
+                        - Removed
+                        pattern: ^(Managed|Unmanaged|Force|Removed)$
+                        type: string
+                      registriesNamespace:
+                        default: rhoai-model-registries
```

Comment:
the biggest part i do not like is this one:  ( i know i am the one suggested for this value i this PR, however i am debating with myself).

@lburgazzoli  @VaishnaviHire .
if we use odh-model-registries as default value here,  we have the same API for odh and rhoai, which should be the case.
But we MIGHT have a problem: user gets a wrong idea of what is the value will be used out of the box or by reading CRD spec.
we can set the rhoai-model-regiestries in "sample" and CSV, still if anyone wants to create DSC CR via cli, it can be "odh-model-registeries" pass down to dashboard.

on the other hand, we support config different namespace. so if someone explicitly set "odh-model-registries" for a rhoai offering, it is acceptable too. 

:thinking: :thinking:  :thinking:

Reply:
Can it use some platform independent name for both versions?

Reply:
i think it was a decision from modelreg team, not we want them to have different name in upstream and downstream.

Reply:
sorry, i was stupid
this should not be a problem.
in API, the default can be the same in ODH and rhoai, "odh-model-regiestries" all we need to do is a different one in sample.

so here should be 
`default: odh-model-registries`

Reply:
No problem. All the entries?